gnd/src/main/java/com/google/android/gnd/repository/ProjectRepository.java

```diff
@@ -176,6 +176,7 @@ public void clearActiveProject() {
 
   public ImmutableList<Layer> getModifiableLayers(Project project, User user) {
     String role = project.getAcl().get(user.getEmail());
+    Timber.v("User %s, Layers: %s", project.getLayers(), user);
```

Comment:
Is there any reason not to include these? Logs get deactivated in release builds, but may help identifying issues in debug builds. Wdyt?

@dturner Is there any official guidance on this?

Reply:
I agree with @gino-m. There's no downside that I'm aware of to leaving logging in, and it can aid debugging and readability. I don't believe there is any official guidance beyond https://developer.android.com/studio/debug/am-logcat. 

In case it's useful, here's the Timber tree setup in one of our most popular sample apps, IOSched: https://github.com/google/iosched/blob/b0c6384b4b26efdac16142b8e85b1d8859053a48/mobile/src/main/java/com/google/samples/apps/iosched/MainApplication.kt#L48

Reply:
Ah ok, great! The main tradeoff imo re: excessive logging is that it hurts readability -- it introduces a lot of noise lines in the source. But it's not the end of the world by any means and it seems like having some runtime debugging available is more important for the moment.

Reply:
There's definitely a trade-off. One possible signal could be whether the log line hurts vs helps readability. This specific one isn't particularly useful, so I think it's ok to remove it.